apps/services/notifications/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,21 @@
-export const environment = {
+import assert from 'assert'
+
+const environment = {
   production: true,
+
+  mainQueueName: process.env.MAIN_QUEUE_NAME,
+  deadLetterQueueName: process.env.DEAD_LETTER_QUEUE_NAME,
+
+  sqsConfig: {
+    region: process.env.SQS_REGION ?? 'eu-west-1',
```

Comment:
I don't really have an opinion on this.
I just see `eu-west-1` hardcoded in a few places in the repo.
Maybe this could be hardcoded as well.
I thought this would be a sinsible default.